mlflow/pyspark/ml/__init__.py

```diff
@@ -1066,7 +1078,7 @@ def _infer_model_signature(input_example_slice):
                         )
                         model_output = None
                     else:
-                        model_output = model_output.toPandas()
+                        model_output = model_output
```

Comment:
We can just remove this else condition, right?

Reply:
oh, good catch.